lib/CMakeLists.txt

```diff
@@ -449,7 +449,7 @@ if(ORC_FOUND)
 
         #create a rule to generate the source and add to the list of sources
         add_custom_command(
-            COMMAND ${ORCC_EXECUTABLE} --include math.h --implementation -o ${orcc_gen} ${orc_file}
+            COMMAND ${ORCC_EXECUTABLE} --include math.h --implementation --include volk/volk_complex.h -o ${orcc_gen} ${orc_file}
```

Comment:
`--include volk/volk_complex.h` is needed so that ORC programs can use VOLK's complex types in their function signatures.